src/modules/contract/components/walletModal/index.tsx

```diff
@@ -32,10 +35,21 @@ export const WalletModal: React.FC<WalletModalProps> = ({ open, handleClose, han
           <div className="font-medium text-sm leading-5 text-Gray-light-mode-700 ">
             To accept this offer you need to connect your wallet
           </div>
-          {isConnected ? <Dapp.Connect /> : <ConnectButton handleClick={() => openConnect()} />}
+          {/* FIXME(Elaine): handleLaceConnect null for now because there's not really a reason with CIP-30 to turn it inside out */}
+          {isConnected ? (
+            <Dapp.Connect />
+          ) : (
+            <ConnectButtons handleWalletConnect={() => openConnect()} handleLaceConnect={() => undefined} />
```

Comment:
handeleLaceConnect={() => undefiend} 
you can pass nothing as a optional prop until later you will make it a mandatory prop